docs/design/colocation/Overview.md

```diff
@@ -0,0 +1,456 @@
+## Motivation
+
+Volcano now supports unified scheduling of microservices and batch workloads, it's compatible with the default scheduling algorithms of kubernetes, and provides rich scheduling policies for microservices and batch jobs, so it is suitable for unified scheduling of different types of jobs in colocation scenarios, and improves resource utilization simultaneously.
+
+## Architecture
+
+Volcano agent supports cloud-native online and offline workloads colocation. It runs as a daemonset in the kubernetes cluster, and is committed to improving the resource utilization of the kubernetes cluster and meanwhile ensures the Qos of online workloads through CPU/Memory/Network resource isolation and suppression.
+
+![architecture](images/architecture.jpg)
+
+### Qos Model
+
+|              Qos              |                 Typical  Usage Scenario                  |
+| :---------------------------: | :------------------------------------------------------: |
+|     LC(Latency Critical)      | Exclusive CPU for latency critical core online workloads |
+| HLS(Highly Latency Sensitive) |       Extremely latency sensitive online workloads       |
+|     LS(Latency Sensitive)     |      Latency sensitive approximate online workloads      |
+|        BE(Best Effort)        |    Offline AI/Big Data workloads, tolerable evictions    |
+
+You can set different qos levels to identify workloads of different priorities, and there is no seperate `priorityClass` definition to be consistent with k8s.
```

Comment:
If QoS is set and priority is set for a workload, which of the following configurations prevails during scheduling?
For example, if workload-1 QoS is LS, priorityClass is 100, workload-2 QoS is LC, and priorityClass is 50, which workload is scheduled first when cluster resources are insufficient?

Reply:
This is compatible with k8s's priority and higher priority pods will be scheduled first and user should set proper priority for different Qos level of pod.

Reply:
In you example, the LC and LS workloads use different kinds of resource, there is no resource competation between them. For the scheduling sequence, it is controlled by priority which is the same as  before.